.github/workflows/ci.yml

```diff
@@ -912,14 +920,15 @@ jobs:
       - name: "Build iree-run-module and test benchmark suite modules"
         run: |
           ./build_tools/github_actions/docker_run.sh \
-            --env "${PLATFORM^^}_ARCH=${ARCHITECTURE}" \
-            --env "IREE_BUILD_${PLATFORM^^}_DIR=build-${PLATFORM}-${ARCHITECTURE}" \
+            --env "IREE_TARGET_PLATFORM=${PLATFORM}" \
+            --env "IREE_TARGET_ARCH=${ARCH}" \
             --env "BUILD_PRESET=benchmark-suite-test" \
             --env "IREE_HOST_BINARY_ROOT=${HOST_BUILD_DIR}/install" \
             --env "E2E_TEST_ARTIFACTS_DIR=${E2E_TEST_ARTIFACTS_DIR}" \
             "${DOCKER_IMAGE}" \
             bash -euo pipefail -c \
-              "${RUN_SCRIPTS}"
+              "${RUN_SCRIPTS}" \
+              "${TARGET_BUILD_DIR}"
```

Comment:
The build_riscv.sh script won't be passed the argument

Reply:
Now I know why we pass arguments with envs. Done.

Reply:
Hmm maybe having env vars as an option was a mistake. It bugs me a bit that we have this explicit `bash -c` stuff. Ah well.